src/main/java/com/example/test/controllers/EmployeeController.java

```diff
@@ -9,23 +9,28 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+
+
 @RestController
 @RequestMapping("/employee")
 @RequiredArgsConstructor
+
 public class EmployeeController {
     @Autowired
     private EmployeeService employeeService;
 
     @GetMapping(path ="{employeeId}")
-    public ResponseEntity<?> getEmployee(@PathVariable Long employeeId) {
-
+    public ResponseEntity<?> getEmployee(@PathVariable Long employeeId){
+        if (employeeId < 1){
+            return new ResponseEntity<>("enter valid number", HttpStatus.BAD_REQUEST);
+        }
```

Comment:
Can you try to use the annotation @Min
https://spring.io/guides/gs/validating-form-input/